ingestion/src/metadata/ingestion/source/database/databricks.py

```diff
@@ -121,7 +133,56 @@ def get_columns(self, connection, table_name, schema=None, **kw):
     return result
 
 
+@reflection.cache
+def get_schema_names(connection, **kw):
+    # Equivalent to SHOW DATABASES
+    _ = connection.execute(f"USE CATALOG {kw.get('database')}")
+    return [row[0] for row in connection.execute("SHOW SCHEMAS")]
+
+
+def get_schema_names_reflection(self, **kw):
+    """Return all schema names."""
+
+    if hasattr(self.dialect, "get_schema_names"):
+        with self._operation_context() as conn:  # pylint: disable=protected-access
+            return self.dialect.get_schema_names(conn, info_cache=self.info_cache, **kw)
+    return []
+
+
+@reflection.cache
+def get_view_definition(
+    self, connection, view_name, schema=None, **kw  # pylint: disable=unused-argument
+):
+    query = f"SHOW CREATE TABLE {kw.get('database')}.{schema}.{view_name}"
+    try:
+        result = connection.execute(query)
+        view_definition = result.fetchone()
+        return view_definition[0] if view_definition else ""
+    except Exception as exc:
+        logger.debug(traceback.format_exc())
+        logger.warning(f"Unexpected exception getting view with query [{query}]: {exc}")
+        return ""
+
+
+def get_view_definition_reflection(self, view_name, schema=None, **kw):
```

Comment:
is it necessary to overwrite this function?

Reply:
Yes, I try out without that function, and it was not working without that.